sw/device/tests/ottf_example_concurrency_test.c

```diff
@@ -0,0 +1,16 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/runtime/log.h"
+#include "sw/device/lib/testing/test_framework/ottf.h"
+
+const test_config_t kTestConfig = {
+    .can_clobber_uart = false,
+    .enable_concurrency = true,
```

Comment:
It will be nice to show two tasks running concurrently in this example. But we can do that later.

Reply:
yes, thats the plan when the FreeRTOS API wrappers are added (see task burndown in #8015)